paddlenlp/transformers/xlnet/tokenizer.py

```diff
@@ -37,44 +37,54 @@
 
 class XLNetTokenizer(PretrainedTokenizer):
     """
-    Constructs an XLNet tokenizer. Based on `SentencePiece <https://github.com/google/sentencepiece>`__.
+    Constructs an XLNet tokenizer based on `SentencePiece <https://github.com/google/sentencepiece>`__.
+
+    This tokenizer inherits from :class:`~paddlenlp.transformers.tokenizer_utils.PretrainedTokenizer`
+    which contains most of the main methods. For more information regarding those methods,
+    please refer to this superclass.
```

Comment:
Maybe add a hyperlink to "this superclass"?

Reply:
thanks, done